tsdb/engine/tsm1/encoding.go

```diff
@@ -70,6 +70,10 @@ func (v Values) MaxTime() int64 {
 }
 
 func (v Values) Encode(buf []byte) ([]byte, error) {
+	if len(v) == 0 {
```

Comment:
This should not be necessary, according to @pauldix. Perhaps it's just for safety, up to you.